fix: case (in)sensitiv guest order validation (#8860)
shopware/platform: changelog/_unreleased/2025-04-28-case-insensitive-guest-order-check.md

---
title: Case insensitive guest order e-mail and postal code check
issue: #8686
---
# Core

> * Changed `\Shopware\Core\Checkout\Order\SalesChannel\OrderRoute::checkGuestAuth`. Compare the guest's e-mail address with `strtolower` and the postal code with `strtoupper` to avoid small typing differences.

Reviewer: ```suggestion
* Changed `\Shopware\Core\Checkout\Order\SalesChannel\OrderRoute::checkGuestAuth` to compare guest email address and postalcode case insensitive.
```
